cherry_picker/cherry_picker/test.py

```diff
@@ -192,57 +258,110 @@ def test_is_not_cpython_repo():
+    relative_config_path = '.cherry_picker.toml'
+    cfg = tmpdir.join(relative_config_path)
     cfg.write('param = 1')
-    assert str(find_config()) == str(cfg)
+    subprocess.run('git add .'.split(), check=True)
+    subprocess.run(('git', 'commit', '-m', 'Initial commit'), check=True)
+    scm_revision = get_sha1_from('HEAD')
+    assert find_config(scm_revision) == scm_revision + ':' + relative_config_path
 
 
 def test_find_config_not_found(tmpdir, cd):
     cd(tmpdir)
     subprocess.run('git init .'.split(), check=True)
-    assert find_config() is None
+    subprocess.run(('git', 'commit', '-m', 'Initial commit', '--allow-empty'), check=True)
+    scm_revision = get_sha1_from('HEAD')
+    assert find_config(scm_revision) is None
+
+
+def test_find_config_not_git(tmpdir, cd):
+    cd(tmpdir)
+    assert find_config(None) is None
 
 
 def test_load_full_config(tmpdir, cd):
     cd(tmpdir)
     subprocess.run('git init .'.split(), check=True)
-    cfg = tmpdir.join('.cherry_picker.toml')
+    relative_config_path = '.cherry_picker.toml'
+    cfg = tmpdir.join(relative_config_path)
     cfg.write('''\
     team = "python"
     repo = "core-workfolow"
     check_sha = "5f007046b5d4766f971272a0cc99f8461215c1ec"
     default_branch = "devel"
     ''')
+    subprocess.run('git add .'.split(), check=True)
+    subprocess.run(('git', 'commit', '-m', 'Initial commit'), check=True)
+    scm_revision = get_sha1_from('HEAD')
     cfg = load_config(None)
-    assert cfg == {'check_sha': '5f007046b5d4766f971272a0cc99f8461215c1ec',
-                   'repo': 'core-workfolow',
-                   'team': 'python',
-                   'fix_commit_msg': True,
-                   'default_branch': 'devel',
-                   }
+    assert cfg == (
+        scm_revision + ':' + relative_config_path,
+        {
+            'check_sha': '5f007046b5d4766f971272a0cc99f8461215c1ec',
+            'repo': 'core-workfolow',
+            'team': 'python',
+            'fix_commit_msg': True,
+            'default_branch': 'devel',
+        },
+    )
 
 
 def test_load_partial_config(tmpdir, cd):
-    cfg = tmpdir.join('.cherry_picker.toml')
+    cd(tmpdir)
+    subprocess.run('git init .'.split(), check=True)
+    relative_config_path = '.cherry_picker.toml'
+    cfg = tmpdir.join(relative_config_path)
     cfg.write('''\
     repo = "core-workfolow"
     ''')
-    cfg = load_config(pathlib.Path(str(cfg)))
-    assert cfg == {'check_sha': '7f777ed95a19224294949e1b4ce56bbffcb1fe9f',
-                   'repo': 'core-workfolow',
-                   'team': 'python',
-                   'fix_commit_msg': True,
-                   'default_branch': 'master',
-                   }
+    subprocess.run('git add .'.split(), check=True)
+    subprocess.run(('git', 'commit', '-m', 'Initial commit'), check=True)
+    scm_revision = get_sha1_from('HEAD')
+    cfg = load_config(relative_config_path)
+    assert cfg == (
+        scm_revision + ':' + relative_config_path,
```

Comment:
Can this be f-stringified? 
```suggestion
        f'{scm_revision}:{relative_config_path}',
```

Reply:
Sure, thanks!